api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/FirstTimeHIVTESTConverter.java

```diff
@@ -0,0 +1,37 @@
+package org.openmrs.module.ugandaemrreports.data.converter;
+
+import org.openmrs.Concept;
+import org.openmrs.Obs;
+import org.openmrs.module.reporting.data.converter.DataConverter;
+import org.openmrs.module.ugandaemrreports.reporting.metadata.Dictionary;
+
+public class FirstTimeHIVTESTConverter implements DataConverter {
+    @Override
+    public Object convert(Object obj) {
+
+        if (obj == null) {
+            return "";
+        }
+        Concept value = ((Obs)obj).getValueCoded();
+        if(value != null && value.equals(Dictionary.getConcept("1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"))) {
+            return "YES";
```

Comment:
@dbaluku This should be Yes and No, not capitals

Reply:
These are to be kept  capital  to have consistency in the report